<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1048</width>
    <height>835</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MARS</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1031</width>
      <height>765</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_37">
     <item>
      <widget class="QPlainTextEdit" name="plainTextEdit">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>$zero</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_33">
           <property name="text">
            <string>0x00000000</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>$at</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_34">
           <property name="text">
            <string>0x00000000</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>$vo</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_35">
           <property name="text">
            <string>0x00000000</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>$v1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_36">
           <property name="text">
            <string>0x00000000</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>$a0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_37">
           <property name="text">
            <string>0x00000000</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>$a1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_38">
           <property name="text">
            <string>0x00000000</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>$a2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_39">
           <property name="text">
            <string>0x00000000</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>$a3</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_40">
           <property name="text">
            <string>0x00000000</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>$t0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_41">
           <property name="text">
            <string>0x00000000</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>$t1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_42">
           <property name="text">
            <string>0x00000000</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>$t2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_43">
           <property name="text">
            <string>0x00000000</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>$t3</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_44">
           <property name="text">
            <string>0x00000000</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_13">
         <item>
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>$t4</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_45">
           <property name="text">
            <string>0x00000000</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_14">
         <item>
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>$t5</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_46">
           <property name="text">
            <string>0x00000000</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_15">
         <item>
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>$t6</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_47">
           <property name="text">
            <string>0x00000000</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_16">
         <item>
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>$t7</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_48">
           <property name="text">
            <string>0x00000000</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_17">
         <item>
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>$s0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_49">
           <property name="text">
            <string>0x00000000</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_18">
         <item>
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>$s1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_50">
           <property name="text">
            <string>0x00000000</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_19">
         <item>
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string>$s2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_51">
           <property name="text">
            <string>0x00000000</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_20">
         <item>
          <widget class="QLabel" name="label_20">
           <property name="text">
            <string>$s3</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_52">
           <property name="text">
            <string>0x00000000</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_21">
         <item>
          <widget class="QLabel" name="label_21">
           <property name="text">
            <string>$s4</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_53">
           <property name="text">
            <string>0x00000000</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_22">
         <item>
          <widget class="QLabel" name="label_22">
           <property name="text">
            <string>$s5</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_54">
           <property name="text">
            <string>0x00000000</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_23">
         <item>
          <widget class="QLabel" name="label_23">
           <property name="text">
            <string>$s6</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_55">
           <property name="text">
            <string>0x00000000</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_24">
         <item>
          <widget class="QLabel" name="label_24">
           <property name="text">
            <string>$s7</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_56">
           <property name="text">
            <string>0x00000000</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_25">
         <item>
          <widget class="QLabel" name="label_25">
           <property name="text">
            <string>$t8</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_57">
           <property name="text">
            <string>0x00000000</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_26">
         <item>
          <widget class="QLabel" name="label_26">
           <property name="text">
            <string>$t9</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_58">
           <property name="text">
            <string>0x00000000</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_27">
         <item>
          <widget class="QLabel" name="label_27">
           <property name="text">
            <string>$k0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_59">
           <property name="text">
            <string>0x00000000</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_28">
         <item>
          <widget class="QLabel" name="label_28">
           <property name="text">
            <string>$k1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_60">
           <property name="text">
            <string>0x00000000</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_29">
         <item>
          <widget class="QLabel" name="label_29">
           <property name="text">
            <string>$gp</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_61">
           <property name="text">
            <string>0x10008000</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_30">
         <item>
          <widget class="QLabel" name="label_30">
           <property name="text">
            <string>$sp</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_62">
           <property name="text">
            <string>0x7fffeffc</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_31">
         <item>
          <widget class="QLabel" name="label_31">
           <property name="text">
            <string>$fp</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_63">
           <property name="text">
            <string>0x00000000</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_32">
         <item>
          <widget class="QLabel" name="label_32">
           <property name="text">
            <string>$ra</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_64">
           <property name="text">
            <string>0x00000000</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1048</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="text">
    <string>Run</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
